frontend/app/src/components/dashboard/VisibleColumnsSelector.vue

```diff
@@ -49,35 +49,45 @@ const onVisibleColumnsChange = async (visibleColumns: TableColumn[]) => {
 
   await store.updateSetting(payload);
 };
+
+const active = (value: TableColumn) =>
+  get(currentVisibleColumns).includes(value);
+
+const update = (value: TableColumn) => {
+  const visible = [...get(currentVisibleColumns)];
+  const index = visible.indexOf(value);
+  if (index === -1) {
+    visible.push(value);
+  } else {
+    visible.splice(index, 1);
+  }
+
+  onVisibleColumnsChange(visible);
+};
 </script>
 
 <template>
-  <VList>
-    <VListItemGroup
-      :value="currentVisibleColumns"
-      multiple
-      @change="onVisibleColumnsChange($event)"
-    >
-      <template v-for="(item, i) in availableColumns">
-        <VListItem :key="i" :value="item.value">
-          <template #default="{ active }">
-            <VListItemContent>
-              <VListItemTitle>
-                {{ item.text }}
-              </VListItemTitle>
-            </VListItemContent>
+  <div class="py-2">
```

Comment:
Wasn't removing this vertical padding on the figma page were Raquel adds feedback? 

I will merge as it is but we should probably remove it later.